Common/BaselineSelection.py

```diff
@@ -40,7 +40,7 @@ def Initialize(loadTF=False, loadHHBtag=False):
             if load_result != 0:
                 raise RuntimeError(f"HHBtagWrapper failed to load with status {load_result}")
             ROOT.gInterpreter.Declare(f'#include "{header_path_HHbTag}"')
-            ROOT.gROOT.ProcessLine(f'HHBtagWrapper::Initialize("{os.environ["CMSSW_BASE"]}/src/HHTools/HHbtag/models/", 2)')
+            ROOT.gROOT.ProcessLine(f'HHBtagWrapper::Initialize("{os.environ["CMSSW_BASE"]}/src/HHTools/HHbtag/models/", 3)')
```

Comment:
To use HHbtag v3 you also need to change the definition of input variables (e.g. channel).